torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -206,6 +211,12 @@ class MixedPrecision:
     # TODO: buffer + param are usually of the same type, if user specifies
     # param but not buffer, should we automatically make buffer be the same?
     buffer_dtype: Optional[torch.dtype] = None
+    # Whether to upcast gradients back to the full parameter precision after
```

Comment:
nit: I am not sure if we want this duplicate comment with the docstring (for maintainability reasons).